j/jjwt/jjwt_ubi_8.5.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : jjwt
+# Version       : master
+# Source repo   : https://github.com/jwtk/jjwt
+# Tested on     : UBI 8.5
+# Language      : Java
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Vathsala . <vaths367@in.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=jjwt
+#PACKAGE_VERSION is configurable can be passed as an argument.
+PACKAGE_VERSION=${1:-master}
```

Comment:
Don't use master..
Use either tag/branch/commit-id.

Reply:
Updated the script with commit-id